draft-yoshino-wish.xml

```diff
@@ -172,6 +172,13 @@
       </t>
 
       <t>
+        The CMP bit is used when a compression is in use.
+        Otherwise, it MUST be set to 0.
+        It indicates whether the message is compressed.
+        An endpoint can choose not to compress a message with incompressible contents and send the message with the CMP bit set to 0.
```

Comment:
ok. this sentence was taken from permessage-deflate spec where non-compressing a message was exceptional. here, we shouldn't use this wording. rephrasing.